javascript/src/URDFLoader.js

```diff
@@ -101,6 +105,8 @@ class URDFLoader {
         const packages = options.packages || '';
         const loadMeshCb = options.loadMeshCb || this.defaultMeshLoader.bind(this);
         const workingPath = options.workingPath || '';
+        const visualEnabled = options.visualEnabled || false;
```

Comment:
Visual nodes should be parsed by default, right? This should be `true`

Reply:
But in that case, if the user explicitly gives a false, it'll fallback to true.
Ideally this should be `true` for falsy values like `undefined` and `false` when it's exactly false.

In this case the additional check is not required since the `Object.assign` in load function ensures it's not undefined. If I remove the option default from the load function, I'll have to explicitly check here. Shall I make that change?

The other thing that I guess could be done is to have
`this.defaultOptions` in the constructor,
`Object.assign({ ...this.defaultOptions }, options)` in load function and
`const { packages, loadMeshCb } = { ...this.defaultOptions, options }` in parse.